XCSummary/Templates/CMHTMLReportBuilder.m

```diff
@@ -137,12 +137,13 @@ - (void)_appendActivity:(CMActivitySummary *)activity indentation:(CGFloat)inden
     if (activity.hasScreenshotData)
     {
         templateFormat = [self _decodeTemplateWithName:ActivityTemplateWithImage];
-        NSString *imageName = [NSString stringWithFormat:@"Screenshot_%@.png", activity.uuid.UUIDString];
+        CMAttachment *attachment = [activity.attachments firstObject];
```

Comment:
Is it ok to take only first attachment? What if an activity has more than one attachment?

Reply:
Well yeah, I thought about that.

We need to modify template to be able to add more than one screenshot, or even a file.

Reply:
Got it, ok then.
One more question. `firstObject` is nullable, let's check for not nil and return early in this case.